src/subcommand/server.rs

```diff
@@ -1172,6 +1184,37 @@ impl Server {
     })
   }
 
+  async fn inscription_children(
+    Extension(page_config): Extension<Arc<PageConfig>>,
+    Extension(index): Extension<Arc<Index>>,
+    Path(inscription_id): Path<InscriptionId>,
+  ) -> ServerResult<Response> {
+    Self::inscription_children_from_pages(
+      Extension(page_config),
+      Extension(index),
+      Path((inscription_id, 0)),
+    )
+    .await
+  }
+
+  async fn inscription_children_from_pages(
+    Extension(page_config): Extension<Arc<PageConfig>>,
+    Extension(index): Extension<Arc<Index>>,
+    Path((inscription_id, page_index)): Path<(InscriptionId, usize)>,
+  ) -> ServerResult<Response> {
+    let _inscription = index
```

Comment:
@raphjaph Wondering if there's better way to check for the validity of an inscription.

Reply:
Do you mean if it exists or if the inscription id is malformed? This will return a 404 if it's not found and a 400 if the id is malformed.

Reply:
Ah I see what you mean. You could delete the `let _inscription....` part and check wheter the `let children...` returns and empty array and do a 404 but then it would be unclear of the parent inscription doesn't exist or just doesn't have any children. So I think this should be fine for now

Reply:
Makes sense!